conf/app.routes

```diff
@@ -60,3 +60,6 @@ POST        /shut-mucr                             controllers.consolidations.Sh
 # Shut a MUCR summary page
 GET         /shut-mucr-summary                     controllers.consolidations.ShutMucrSummaryController.displayPage()
 POST        /shut-mucr-summary                     controllers.consolidations.ShutMucrSummaryController.submit()
+
+# Submissions page
+GET         /movements                             controllers.ViewSubmissionsController.displayPage()
```

Comment:
The submissions page shows consolidations too though doesnt it?
Should it not just be `/submissions`?